cmd/crossplane/core/core.go

```diff
@@ -56,6 +59,10 @@ func (c *Command) Run(log logging.Logger) error {
 		return errors.Wrap(err, "Cannot create manager")
 	}
 
+	if err := crds.AddToScheme(mgr.GetScheme()); err != nil {
+		return errors.Wrap(err, "Cannot add Kubernetes APIs to scheme")
```

Comment:
Nitpick: this made me think we were adding the core Kubernetes API types here, which I believe is done automatically as part of the `controller-runtime` machinery. We are just adding CRD and CRDList kind here right?

Reply:
Kubebuilder generated code [adds the core types](https://github.com/muvaf/kubebuilder-init/blob/master/main.go#L39) via client-go scheme. Here, only CRD and CRDList kinds are added from their actual package.

Reply:
Yep, could we adjust this error message to reflect that?

Reply:
Oh I didn't realize that. Fixed. Thanks for catching!